### ISSUE TITLE
Survivalfly: bunnyhopping on slime blocks

### ISSUE THREAD

**Issue by Lysandr0** [Updated-NoCheatPlus/NoCheatPlus#4]
### Full output of /ncp version
```
[INFO] ---- Version information ----

[INFO] #### Server ####

[INFO] git-Spigot-2040c4c-893ad93 ~MC: 1.15.2~

[INFO] detected: 1.15.2

[INFO] #### NoCheatPlus ####

[INFO] Plugin: 3.17-SNAPSHOT-b1153-CustomFork

[INFO] MCAccess: 1.13.2-1.15|? / Bukkit-API

[INFO] Features:

[INFO] blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13 | BlocksMC1_14 | BlocksMC1_15 | MCAccessBukkitModern

[INFO] checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency

[INFO] defaults: pvpKnockBackVelocity

[INFO] packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | WrongTurnAdapter | NoSlow | Fight

[INFO] Hooks: AllViolations~NCP~ 1.0 | ViolationFrequency~NCP~ 1.0

[INFO] #### Related Plugins ####

[INFO] ProtocolLib v4.5.0 | ProtocolSupport v4.29-dev
```
### Describe the issue
Sprintjumping on slime blocks will result in false positives (hSpeed)

### Extra links/Videos (Including debug logs)
[slime_bunnyhop.log](https://github.com/Updated-NoCheatPlus/NoCheatPlus/files/4535135/slime_bunnyhop.log)

https://www.dropbox.com/s/lykee9uvtr9dws4/bunnyhop_slime.mov?dl=0

### Any possible config options changed or plugins that may cause interference?
Default config

**Comment by CaptainObvious0** [comment]
Issue has been fixed with commit 4666291

**Comment by Lysandr0** [comment]
This one should be reopened, issue's back with build n.54.
[bouncebunny.log](https://github.com/Updated-NoCheatPlus/NoCheatPlus/files/4814222/bouncebunny.log)
